### PR TITLE
C2LC-453: Move scene controls that overlap the character.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-453](https://issues.fluidproject.org/browse/C2LC-453) for details.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -135,14 +119,15 @@ body {
...
-    grid-template-rows: 1fr min-content;
+    grid-template-rows: 1fr min-content min-content;
     background-color: #CCE6E2;
 }
 
 .App__world-selector {
     display: flex;
     flex-direction: column;
     margin-top: 1rem;
+    border-bottom: 1px solid #88C3BC;
```
Comment (chosww) [inline src/App.scss:130]: We need to update this border colour in different themes.

Dark Theme: #33717F
Grayscale Theme: #9DA4AF
Contrast Theme: #FFFFFF

File: src/RefreshButton.scss [hunk 2]
```diff
@@ -1,15 +1,57 @@
...
+        path {
+            fill: black;
+        }
+        path.inner-circle {
+            fill: #EA7362;
+        }
+    }
+
+    &:focus {
+        box-shadow: 0 0 0 $c2lc-focus-indicator-width $c2lc-focus-indicator-color;
```
Comment (chosww) [inline src/RefreshButton.scss:21]: The colour we are using for c2lc-focus-indicator-color is #000000, and I think we should update that variable to also use #1E1E1E.

File: src/Themes.scss [hunk 3]
```diff
@@ -2085,6 +2062,44 @@ body.contrast-theme {
...
     }
 
+
+    .RefreshButton {
+        svg {
+            path:not(.refresh-icon) {
+                fill: white;
+            }
+            circle {
+                fill:black;
```
Comment (chosww) [inline src/Themes.scss:2072]: Fill: #1E1E1E;

File: src/Themes.scss [hunk 4]
```diff
@@ -2085,6 +2062,44 @@ body.contrast-theme {
...
+
+            svg {
+                .refresh-icon {
+                    fill: white;
+                }
+                path.inner-circle {
+                    fill: #1E1E1E;
+                }
+                circle {
+                    fill: black;
```
Comment (chosww) [inline src/Themes.scss:2089]: fill: #1E1E1E;

File: src/App.scss [hunk 5]
```diff
@@ -103,27 +103,11 @@ body {
...
-    position: absolute;
-    z-index: 1;
-    bottom: 2rem;
-    right: 2rem;
-    display: grid;
-    grid-template-columns: 1fr;
+    min-width: 19.5rem; 
 }
 
 .App__scene-controls-group {
```
Comment (chosww) [inline src/App.scss:109]: Now this class is only used for the pen toggle switch, might be a good idea to change to more appropriate name?

File: src/App.scss (hunk 1 above)
Comment (the-t-in-rtf) [reply src/App.scss:130]: Good catch, and fixed.

File: src/Themes.scss (hunk 4 above)
Comment (the-t-in-rtf) [reply src/Themes.scss:2089]: Fixed.

File: src/RefreshButton.scss (hunk 2 above)
Comment (the-t-in-rtf) [reply src/RefreshButton.scss:21]: Fixed.

File: src/RefreshButton.scss [hunk 6]
```diff
@@ -1,15 +1,57 @@
...
     stroke: none;
 }
 
+.RefreshButton {
+    background-color: inherit;
+    margin: auto 0;
+
+    svg {
+        path {
+            fill: black;
```
[the remark this answers is not in the document]
Comment (the-t-in-rtf) [reply src/RefreshButton.scss:13]: Fixed.

File: src/App.scss (hunk 5 above)
Comment (the-t-in-rtf) [reply src/App.scss]: Agreed, I changed it to `App__ PenDownToggleSwitch-container`.

**Pull request merged**